src/Microsoft.OData.Edm/Schema/EdmModelBase.cs

```diff
@@ -23,6 +23,11 @@ public abstract class EdmModelBase : EdmElement, IEdmModel
         private readonly Dictionary<string, IEdmTerm> termDictionary = new Dictionary<string, IEdmTerm>();
         private readonly Dictionary<string, IList<IEdmOperation>> functionDictionary = new Dictionary<string, IList<IEdmOperation>>();
 
+        /// <summary>
+        /// Cache of operations that are bindable to entity types.
+        /// </summary>
+        private readonly Dictionary<string, IList<IEdmOperation>> bindableOperationsCache;

```

Comment:
>Dictionary [](start = 25, length = 10)

If we are adding to this dynamically, based on usage of the model, we probably need to make it a concurrent dictionary.  The other dictionaries don't need to be concurrent because they are populated before the model is consumed.

Reply:
I guess the multithreading scenario is handled using ReferenceEquityComparer here. So do we need to use Concurrent dict, or rather just use concurrent dict and remove ReferenceEquityComparer unless there is some other intention